### PR TITLE
[jsk_topic_tools] Add passthrough_nodelet documentation

### PR DESCRIPTION
**Pull request by 708yamaguchi**
I add documentation for `passthrough_nodelet`.

@mqcmd196

### CODE REVIEW

File: doc/jsk_topic_tools/lib/passthrough_nodelet.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+  Start topic relay
+
+* `~stop` (`std_srvs/Empty`)
+
+  Stop topic relay
+
+* `~request_duration` (`jsk_topic_tools/PassthroughDuration`)
+
+  Perform topic relay for a specified duration
```
Comment (pazeshun) [inline doc/jsk_topic_tools/lib/passthrough_nodelet.md:29]: Could you add information like `Duration 0 means infinite relay`?
https://github.com/jsk-ros-pkg/jsk_common/blob/2.2.11/jsk_topic_tools/src/passthrough_nodelet.cpp#L88

File: doc/jsk_topic_tools/lib/passthrough_nodelet.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## Parameters
+
+* `~default_duration` (`double`, default: `10.0`)
+
+  Duration [s] to relay the topic.
```
Comment (pazeshun) [inline doc/jsk_topic_tools/lib/passthrough_nodelet.md:35]: Could you add information like `Duration 0 means infinite relay`?
https://github.com/jsk-ros-pkg/jsk_common/blob/2.2.11/jsk_topic_tools/src/passthrough_nodelet.cpp#L88

**Review by pazeshun — DISMISSED** [review]

File: doc/jsk_topic_tools/lib/passthrough_nodelet.md (hunk 1 above)
Comment (708yamaguchi) [reply doc/jsk_topic_tools/lib/passthrough_nodelet.md]: Thank you for your advice. I updated.

File: doc/jsk_topic_tools/lib/passthrough_nodelet.md (hunk 2 above)
Comment (708yamaguchi) [reply doc/jsk_topic_tools/lib/passthrough_nodelet.md]: Thank you for your advice. I updated.

File: doc/jsk_topic_tools/lib/passthrough_nodelet.md (hunk 1 above)
Comment (pazeshun) [reply doc/jsk_topic_tools/lib/passthrough_nodelet.md]: Thank you!

File: doc/jsk_topic_tools/lib/passthrough_nodelet.md (hunk 2 above)
Comment (pazeshun) [reply doc/jsk_topic_tools/lib/passthrough_nodelet.md]: Thank you!

**Review by pazeshun — APPROVED** [review]
Travis passed!!!

**Comment by pazeshun** [comment]
@k-okada Could you review this?

**Pull request merged**